### PR TITLE
Marais/edit makefile

### PR DESCRIPTION
**Pull request by GaelleMarais**
I added `-lncurses` to compile the ui that is using ncurses.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ build/settings.o: ./src/settings.c ./include/settings.h
 	$(COMPILE)	$< -o $@
 
 build/ui.o: ./src/ui.c ./include/ui.h
-	$(COMPILE)	$< -o $@
+	$(COMPILE)	$< -lncurses -o $@
```
Comment (Kagamihime) [inline Makefile]: I am not sure if you really need to add it here too

Comment (GaelleMarais) [reply Makefile]: Do you want me to delete it?

Comment (Kagamihime) [reply Makefile]: Yes: `-lncurses` is an instruction for the linker, so you don't need it when compiling

Comment (GaelleMarais) [reply Makefile]: Okey done :)

**Comment by Kagamihime** [comment]
You didn't run coala locally, so Travis fails.
And do not just add commits when you are requested changes, amend or rebase your commits instead (you can easily do it with what I showed you last time). Otherwise, the commit history will be messy

**Comment by GaelleMarais** [comment]
Sorry, I try to edit the commits with `git rebase -i master` like you showed me but it looks like it doesn't change the files

**Comment by Kagamihime** [comment]
If you can't make it work, you can still delete your commits with `git reset --soft master` (don't worry, the changes you've made will be put back in your working copy), do your commits again and force push

**Comment by GaelleMarais** [comment]
ok done

**Pull request merged**